static/scss/catalog/body.scss

```diff
@@ -116,21 +129,24 @@ h1, h2, h3, h4, h5 {
         margin: 0;
         padding: 0;
         color: #909090;
+        font-size: 20px;
+        line-height: 26px;
+        font-weight: 500;
+
+        strong {
+          font-weight: 600;
+        }
 
         li {
           padding-bottom: 10px;
-
-          strong {
-            font-weight: 500;
-          }
         }
       }
     }
 
     .offering-type {
       label {
         display: inline-block;
-        padding: 7px;
+        padding: 0 10px;
         border: 1px solid #dedede;
```

Comment:
Could you please convert the variable into a colour if you think it might be reused later on?